### DISCUSSION TITLE
Bad support response times

### DISCUSSION THREAD

**Discussion by mr-nv** [community/community#55579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, on the 11th of April I made a support ticket regarding our organization, which has been hidden from the public. After 34 days I still did not get a reply.

What is going on there? [Shouldn't this be not an issue after 2+ years?](https://github.com/orgs/community/discussions/23989)

**Comment by npiombo** [comment]
Similarly, I put in a request for non-profit verification for my org on March 30th and still waiting, while I update the ticket weekly waiting for any sort of response.

**Reply by veganrav** [reply]
Did you get a response? I'm currently in the same boat

**Comment by Etzshally** [comment]
Response #20353
